test/search.cpp

```diff
@@ -39,4 +75,30 @@ TEST(Search, searchByTitle)
   ASSERT_EQ(mainEntry.getPath(), search.begin().get_url());
 }
 
+TEST(Search, indexFullPath)
+{
+  TempZimArchive tza("testZim");
+  zim::writer::Creator creator;
+  creator.configIndexing(true, "en");
+  creator.startZimCreation(tza.getPath());
+
+  auto item = std::make_shared<TestItem>("testPath", "Test Article", "This is a test article");
+  creator.addItem(item);
+
+  creator.setMainPath("testPath");
+  creator.addMetadata("Title", "Test zim");
+  creator.finishZimCreation();
+
+  zim::Archive archive(tza.getPath());
+
+  zim::Search search(archive);
+  search.set_suggestion_mode(false);
+  search.set_query("test article");
+  search.set_range(0, archive.getEntryCount());
+  search.set_verbose(true);
+
+  // URL should have the namespace prefix "C/"
+  ASSERT_EQ(search.begin().get_url().substr(0, 2), "te");
```

Comment:
We the comment says it must start with "C/" and you test with "te" ?

It is not a problem that new added tests are failing.
In fact, this is a good thing at it ensure that the following correction actually fix a tested feature.